### PR TITLE
7 new episode logistic regression

### LINKED ISSUE #7: New episode: logistic regression
Create logistic regression episode using UKHSA provided data

Reference/update: https://stats.oarc.ucla.edu/r/dae/logit-regression/

### PR DESCRIPTION
**Pull request by quirksahern**
Lesson content and challenges added using the UKHSA provided data

### CODE REVIEW

**Comment by quirksahern** [comment]
update needed to config.yml

File: episodes/26-log-regression.Rmd [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+We can see that dates are currently stored as the char data type. We also do not know the age of the participant when the sample was taken.
+
+```{r}
+
+# Calculate age (in years) as of their last birthday and add as an additional variable to our data.
+
+amrData <- amrData %>% 
+  mutate(
+    age_years_sd = (dob %--% spec_date) %/% years(1)
```
Comment (milanmlft) [inline episodes/26-log-regression.Rmd:68]: Are the `%--%` and `%/%` syntax bits intentional? I've never seen this kind of syntax before, do we cover this in one of the previous episodes?

**Review by milanmlft — APPROVED** [review]
Looks good! Just a small remark about the funky `%*%` syntax

File: episodes/26-log-regression.Rmd (hunk 1 above)
Comment (quirksahern) [reply episodes/26-log-regression.Rmd:68]: Comment added to explain syntax and that it for lubridate

**Comment by quirksahern** [comment]
@milanmlft  I dont know how to resolve this, can you take a look please?

**Comment by quirksahern** [comment]
@milanmlft  can you please take a look at the conflicts in: renv/profiles/lesson-requirements/renv.lock

I don't know what most of it relates to

**Pull request merged**